src/main/java/seedu/address/model/insurance/LifeInsurance.java

```diff
@@ -255,9 +255,7 @@ public boolean equals(Object other) {
         //TODO: Increase the validity of equals
         return other == this // short circuit if same object
                 || (other instanceof LifeInsurance // instanceof handles nulls
-                && ((LifeInsurance) other).premiumString.equals(this.premiumString)
-                && ((LifeInsurance) other).signingDate.isEqual(this.signingDate)
-                && ((LifeInsurance) other).expiryDate.isEqual(this.expiryDate)); // state check
+                && ((LifeInsurance) other).getInsuranceName().equals(this.insuranceName)); // state check
     }
```

Comment:
So if the two insurances have the same name they are equal?

Reply:
hmm i realise this is an attempt to pacify the selection thingy below, ill have to redo this part

Reply:
hmm actually the uuid is perfect for this setting but it is not available in this top-down design as it is not part of the insurance class

Reply:
Okay. I assume you will do it in a separate branch?

Reply:
Merging after receiving your response!

Reply:
ok ill make a commit for that now

Reply:
im still going omit the comparison for owner insured and beneficiary because you havent overrided the equals method there and idk how it should be done